apps/platform-app/src/app/[locale]/(profile)/_workflows/prompts/evaluateProfileNode.ts

```diff
@@ -4,28 +4,99 @@ import {
+AI:
+Reject the profile. Reason: The BIO contain offensive words.
+
+
+User:
+<Name> Camille Moreau
+<BIO> A highly skilled software developer with extensive experience in designing, implementing, and testing cutting-edge applications. Proficient in a variety of programming languages such as JavaScript, Python, and Java.
+<Avatar description> The image depicts a red and black flag with a white circle in the center, featuring a swastika symbol. The flag's design is divided into four sections by two black lines that intersect at the center, where the swastika 
+is located. The swastika itself is a black symbol with four arms of equal length, arranged in a cross-like pattern. The background of the flag is 
+a deep red color, which provides a striking contrast to the white circle 
+and black swastika. Overall, the image presents a bold and eye-catching design that is likely intended to convey a sense of power or authority.   
+AI:
+Reject the profile. Reason: The photo contains a political symbol (swastika) which is considered an offensive element
+
+User:
+<Name> Karen Kovalsky
+<BIO> With over 50 years of hands-on experience in software development, I have honed my skills in various programming languages including Java, Python, and C++. My expertise lies in designing, implementing, and maintaining robust and scalable applications that meet the highest industry standards.
+<Avatar description> The image depicts a man with dark hair, gazing directly at the camera. His hair is styled in a manner that suggests it may be wet or damp, and he 
+appears to be shirtless. The background of the image is predominantly white, with a subtle shadow cast by the man's head and shoulders. Notably, there are no other individuals or objects visible in the image.
+AI:
+Send for manual verfication. Reason: An incredible amount of years of experience.    
+
+User:
+<Name> Dawid Florian
+<BIO> With a solid foudation in software enginering and over seven yaers of professinal experiance, I excells in deve high-performance webb aplications. My proficency spans accross multiple programing languages, including JavaSscript, Pyton, and PLP, allowing me to delivver inovative sotutioons that drivve buisness sucses. Know for my atention to detale and comitment to writting clean, maintainableecode, I continuosly strive to improve and adapt to new techologies in the fast-paced tech industy
+<Avatar description> The image depicts a man with dark hair, gazing directly at the camera. His hair is styled in a manner that suggests it may be wet or damp, and he 
+appears to be shirtless. The background of the image is predominantly white, with a subtle shadow cast by the man's head and shoulders. Notably, there are no other individuals or objects visible in the image.
+AI:
+Reject the profile. Reason: Multiple serious language errors    
+
 `)
```

Comment:
**nitpick**: Ensure consistent formatting by aligning the colons in the template.